### ISSUE TITLE
Update to Selenium 3.7

### ISSUE THREAD

**Issue by pbolduc** [ipjohnson/SeleniumFixture#10]
Updated project references to target latest Selenium version.

**Comment by ipjohnson** [comment]
I've created a new branch for this work and done an initial checkin (it compiles but need to do testing)

**Comment by ipjohnson** [comment]
Nuget feed for nightly build
https://ci.appveyor.com/nuget/seleniumfixture-nightly

**Comment by ipjohnson** [comment]
@pbolduc I'm interested in doing an pre-release of this to nuget. Do you have any other issues/changes you'd like to see in it before pushing?

**Comment by pbolduc** [comment]
No I do not.

**Comment by ipjohnson** [comment]
Ok I've pushed an alpha version to nuget

**Comment by ipjohnson** [comment]
@pbolduc were you able to do any testing on this version?

**Comment by pbolduc** [comment]
TLDR: No.

I ran into some initial problems, but I think it was a problem on my side. I haven't been working on the Selenium tests since last fall.
